### ISSUE TITLE
Issue with "Using the aria-describedby attribute": …

### ISSUE THREAD

**Issue by wmcb91** [mdn/content#740]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute

#### What information was incorrect, unhelpful, or incomplete?
The link to the aria-labelledby docs is incorrect and returns a 404.

#### Specific section or headline?
The broken link is in the first paragraph where the text reads "This is very similar to [aria-labelledby:](https://developer.mozilla.org/en-US/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute)..." 

#### What did you expect to see?
The link to go to the correct page.

#### Did you test this? If so, how?
I clicked the link and was brought to a 404 page. I compared the URL to the actual URL of the aria-labelledby docs and saw the difference (the current link is missing /docs/web/).

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/aria/aria_techniques/using_the_aria-describedby_attribute`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute
* GitHub URL: https://github.com/mdn/content/blob/master/files/en-us/web/accessibility/aria/aria_techniques/using_the_aria-describedby_attribute/index.html
* Report started: 2020-12-29T21:48:32.982Z

</details>

**Comment by wmcb91** [comment]
PR created: https://github.com/mdn/content/pull/741

**Comment by wmcb91** [comment]
Fixed in https://github.com/mdn/content/pull/722
